server/src/main/resources/org/elasticsearch/common/reference-docs-links.txt

```diff
@@ -1,48 +1,48 @@
-BOOTSTRAP_CHECKS                                                bootstrap-checks.html
-BOOTSTRAP_CHECK_ENCRYPT_SENSITIVE_DATA                          bootstrap-checks-xpack.html#bootstrap-checks-xpack-encrypt-sensitive-data
-BOOTSTRAP_CHECK_PKI_REALM                                       bootstrap-checks-xpack.html#bootstrap-checks-xpack-pki-realm
-BOOTSTRAP_CHECK_ROLE_MAPPINGS                                   bootstrap-checks-xpack.html#bootstrap-checks-xpack-role-mappings
-BOOTSTRAP_CHECK_TLS                                             bootstrap-checks-xpack.html#bootstrap-checks-tls
-BOOTSTRAP_CHECK_TOKEN_SSL                                       bootstrap-checks-xpack.html#bootstrap-checks-xpack-token-ssl
-CONTACT_SUPPORT                                                 troubleshooting.html#troubleshooting-contact-support
-UNASSIGNED_SHARDS                                               red-yellow-cluster-status.html
-EXECUTABLE_JNA_TMPDIR                                           executable-jna-tmpdir.html
-NETWORK_THREADING_MODEL                                         modules-network.html#modules-network-threading-model
-ALLOCATION_EXPLAIN_API                                          cluster-allocation-explain.html
-NETWORK_BINDING_AND_PUBLISHING                                  modules-network.html#modules-network-binding-publishing
-SNAPSHOT_REPOSITORY_ANALYSIS                                    repo-analysis-api.html
-S3_COMPATIBLE_REPOSITORIES                                      repository-s3.html#repository-s3-compatible-services
-LUCENE_MAX_DOCS_LIMIT                                           size-your-shards.html#troubleshooting-max-docs-limit
-MAX_SHARDS_PER_NODE                                             size-your-shards.html#troubleshooting-max-shards-open
-FLOOD_STAGE_WATERMARK                                           fix-watermark-errors.html
-X_OPAQUE_ID                                                     api-conventions.html#x-opaque-id
-FORMING_SINGLE_NODE_CLUSTERS                                    modules-discovery-bootstrap-cluster.html#modules-discovery-bootstrap-cluster-joining
-CIRCUIT_BREAKER_ERRORS                                          circuit-breaker-errors.html
-ALLOCATION_EXPLAIN_NO_COPIES                                    cluster-allocation-explain.html#no-valid-shard-copy
-ALLOCATION_EXPLAIN_MAX_RETRY                                    cluster-allocation-explain.html#maximum-number-of-retries-exceeded
-SECURE_SETTINGS                                                 secure-settings.html
-CLUSTER_SHARD_LIMIT                                             misc-cluster-settings.html#cluster-shard-limit
+INITIAL_MASTER_NODES                                            deploy-manage/deploy/self-managed/important-settings-configuration#initial-master-nodes
+DISCOVERY_TROUBLESHOOTING                                       troubleshoot/elasticsearch/discovery-troubleshooting
```

Comment:
If my earlier suggestion is feasible, the contents of this file would look like this

> DISCOVERY_TROUBLESHOOTING                                       https://www.elastic.co/docs/troubleshoot/elasticsearch/discovery-troubleshooting

or if necessary:

> DISCOVERY_TROUBLESHOOTING                                       https://www.elastic.co/docs/troubleshoot/elasticsearch/discovery-troubleshooting?version=9.0